code/modules/clothing/chameleon.dm

```diff
@@ -217,7 +217,25 @@
 	chameleon_action = new(src)
 	chameleon_action.chameleon_type = /obj/item/clothing/under
 	chameleon_action.chameleon_name = "Jumpsuit"
-	chameleon_action.chameleon_blacklist = typecacheof(list(/obj/item/clothing/under, /obj/item/clothing/under/color, /obj/item/clothing/under/rank), only_root_path = TRUE)
+	chameleon_action.chameleon_blacklist = typecacheof(list(
```

Comment:
is this a hard blacklist or does this also affect the subtypes?

Reply:
Since it has `only_root_path = TRUE` I would assume it does not affect subtypes. If I understand that correctly that is.

Reply:
could you just confirm in-game that at least one subtype of one of these types appears? After that I'll slap an approval on it cause the code looks fine.

Reply:
I'll do you one better, a jumpsuit of every subtype from these

![image](https://user-images.githubusercontent.com/71735193/197353531-cc17af36-35b0-44da-8745-a610f192fe0a.png)

![image](https://user-images.githubusercontent.com/71735193/197353593-e1abb064-4f22-4227-9bf0-4cdeac97fcb9.png)



Changed it to `only_root_path = FALSE` just to check what would happen in testing, the window to change the chemeleon didn't even appear.